packages/gestalt/src/TextArea.js

```diff
@@ -221,6 +221,7 @@ const TextAreaWithForwardRef: AbstractComponent<Props, HTMLTextAreaElement> = fo
 
   const tagsWrapperStyle = {
     minHeight: rows * ROW_HEIGHT + INPUT_PADDING_WITH_TAGS,
```

Comment:
I'm curious about why we need both minHeight & maxHeight being the same number, aint that just the same as height?

A follow up would be calculating the space that a row of tag + top/bottom margin takes and add ~1/3 of the next row. I think we should show rows + 1/3 of the next. But that's a design design too. We can iterate on that.